src/types/Profiles.cpp

```diff
@@ -44,11 +52,12 @@ void Profiles::generateRandomPerfValues(unsigned long int seed, int lower_bound,
     throw std::invalid_argument(
         "Lower bound must be lower than the upper bound.");
   }
-  srand(seed);
+  std::random_device rd;
```

Comment:
If we use time(0) as default value in the random generator, we won't need this random device anymore !